R/info_tables.R

```diff
@@ -327,43 +327,47 @@ info_currencies <- function(
     } else {
       curr <- currencies
     }
+    tab_1 <- curr
+    tab_1$symbol <- NULL
+    tab_1$value <- 49.95
+    tab_1 <- dplyr::relocate(tab_1, "curr_name")
+    tab_1 <- gt(tab_1)
 
-    tab_1 <-
-      curr %>%
-      dplyr::select(-symbol) %>%
-      dplyr::select(curr_name, dplyr::everything()) %>%
-      dplyr::mutate(value = 49.95) %>%
-      gt()
-
-    for (i in seq(nrow(curr))) {
+    for (i in seq_len(nrow(curr))) {
 
       tab_1 <-
-        tab_1 %>%
         fmt_currency(
```

Comment:
Not convinced by this fiddling, will it make things harder to expand if needed? I can revert if you want!